### PR TITLE
#5201 fix dropdown

### LINKED ISSUE #5201: Sort by checkmark jumping to second row
image
Sort by checkmark jumping to second row
- we can I think also replace the checkmark icon with fa icon as this one is afaik imported custom svg

### PR DESCRIPTION
**Pull request by prachi00**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇 \_ Let's make a quick check before the contribution.

## PR Type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

## Context

- [x] Closes #5201
- [ ] Requires deployment <>

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

#### Optional

- [ ] I've tested it at </bsx/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Had issue bounty label?

- [x] Fill up your KSM address: 
[Payout](https://beta.kodadot.xyz/transfer/?target=EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.
Screen Shot 2023-03-08 at 7 37 36 PM

### CODE REVIEW

**Review by vikiival — CHANGES_REQUESTED** [review]
Smol by @preschian

File: components/explore/ExploreSort.vue [hunk 1]
```diff
@@ -33,16 +32,14 @@
...
         <span>
           {{
             $i18n.t(isItems ? `sort.${option}` : `sort.collection.${option}`)
           }}
         </span>
-        <img
-          v-if="selectedSort.includes(option)"
-          class="sort-check"
-          src="/checkmark.svg" />
+        <b-icon v-if="selectedSort.includes(option)" icon="check" />
```
[the remark this answers is not in the document]
Comment (prachi00) [reply components/explore/ExploreSort.vue:42]: @preschian this could be done in #5216

File: libs/ui/src/components/NeoDropdown/NeoDropdown.scss [hunk 2]
```diff
@@ -29,8 +29,8 @@ $dropdown-item-hover-background-color: #ffe5f3;
   &.is-active,
   &--active {
     @include ktheme() {
-      color: theme('text-color-inverse');
-      background: theme('background-color-inverse');
+      color: theme('text-color-inverse')!important;
+      background: theme('background-color-inverse')!important;
```
Comment (daiagi) [inline libs/ui/src/components/NeoDropdown/NeoDropdown.scss:33]: is the !important really necessary?

Comment (prachi00) [reply libs/ui/src/components/NeoDropdown/NeoDropdown.scss:33]: yeah, without it, hover states were getting applied, when it shouldnt

**Comment by prachi00** [comment]
> 
Screen Shot 2023-03-11 at 3 25 35 PM

File: components/explore/ExploreSort.vue (hunk 1 above)
[the remark this answers is not in the document]
Comment (prachi00) [reply components/explore/ExploreSort.vue]: done

**Review by daiagi — APPROVED** [review]

File: components/explore/ExploreSort.vue [hunk 3]
```diff
@@ -33,16 +32,15 @@
...
         <span>
           {{
             $i18n.t(isItems ? `sort.${option}` : `sort.collection.${option}`)
           }}
         </span>
-        <img
-          v-if="selectedSort.includes(option)"
-          class="sort-check"
-          src="/checkmark.svg" />
+        <o-icon v-if="selectedSort.includes(option)" icon="check" class="ml-2">
```
Comment (vikiival) [inline components/explore/ExploreSort.vue:42]: @preschian dont we have some neoicon component?

Comment (prachi00) [reply components/explore/ExploreSort.vue:42]: @vikiival not yet, I added it in #5237

Comment (vikiival) [reply components/explore/ExploreSort.vue:42]: https://github.com/kodadot/nft-gallery/pull/5237 is merged Please proceed

**Review by vikiival — CHANGES_REQUESTED** [review]
Please use NeoIcon

**Review by vikiival — APPROVED** [review]

**Comment by vikiival** [comment]
pay 20 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $20 @ 33.98 USD/KSM ~ 0.589 $KSM
🧗 [EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU](https://kusama.subscan.io/account/EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)
🔗 [0x0308b7badd1f6534bc3301ea3c4a09e820b6622f8a08387c4b9a85ed7427fb46](https://kusama.subscan.io/extrinsic/0x0308b7badd1f6534bc3301ea3c4a09e820b6622f8a08387c4b9a85ed7427fb46)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**